### PR TITLE
Sunset google analytics

### LINKED ISSUE #550: Sunsetting the Google Analytics services
Per the discussion in #546, we are temporarily trying both google analytics and plausible.  Once we decide whether to use google analytics or plausible, we will need to either:

## Remove google analytics (see #549)

Remove the following files
```
layouts/_default/baseof.html
layouts/partials/meta.html
```
as well as this line
```
googleAnalytics: UA-162151853-1
```
in `config.yaml.in`.

## Remove plausible analytics (see #546)

Remove
```
  plausible:
    dataDomain: numpy.org
```
from  `config.yaml.in` and
```
## Analytics

A self-hosted version of [Plausible.io](https://plausible.io) is used to gather simple
and privacy-friendly analytics for the site. The dashboard can be accessed
[here](https://views.scientific-python.org/numpy.org).
```
from `README.md`.

## Why use plausible?

The theme will only support plausible going forward, so using it will involve less customization for numpy.org.  For more information, see https://plausible.io/vs-google-analytics.

### PR DESCRIPTION
**Pull request by jarrodmillman**
Close #550

### CODE REVIEW

**Review by InessaPawson — APPROVED** [review]
@jarrodmillman Looks good. Thank you for taking care of this!

File: layouts/_default/baseof.html [hunk 1]
```diff
@@ -1,47 +0,0 @@
...
-    <!-- Render custom header -->
-    {{ partial "head.html" . }}
-
-    <!-- Refer: https://regisphilibert.com/blog/2018/08/hugo-multilingual-part-1-managing-content-translation/ -->
-    {{ if .IsTranslated }}
-      {{ range .Translations }}
-      <link rel="alternate" hreflang="{{ .Language.Lang }}" href="{{ .RelPermalink }}" title="{{ .Language.LanguageName }}" />
-      {{ end }}
-    {{ end }}
-    {{ template "_internal/twitter_cards.html" . }}
```
Comment (rgommers) [inline layouts/_default/baseof.html:19]: Just checking: we don't need this anymore? It's unrelated to analytics, and IIRC this was a deliberate fix at some point.

Comment (jarrodmillman) [reply layouts/_default/baseof.html:19]: Looking into it. If needed, I will fix this in the theme.

Comment (jarrodmillman) [reply layouts/_default/baseof.html:19]: https://github.com/numpy/numpy.org/pull/345

Comment (jarrodmillman) [reply layouts/_default/baseof.html:19]: See #590 .

**Pull request merged**